src/hydra/parseHydraDocumentation.js

```diff
@@ -190,157 +199,203 @@ function findRelatedClass(docs, property) {
 export default function parseHydraDocumentation(entrypointUrl, options = {}) {
   entrypointUrl = removeTrailingSlash(entrypointUrl);
 
-  return fetchEntrypointAndDocs(entrypointUrl, options).then(
-    ({ entrypoint, docs, response }) => {
-      const resources = [],
-        fields = [],
-        operations = [];
-      const title = get(
-        docs,
-        '[0]["http://www.w3.org/ns/hydra/core#title"][0]["@value"]',
-        "API Platform"
-      );
-
-      const entrypointType = get(entrypoint, '[0]["@type"][0]');
-      if (!entrypointType) {
-        throw new Error('The API entrypoint has no "@type" key.');
-      }
-
-      const entrypointClass = findSupportedClass(docs, entrypointType);
-      if (
-        !Array.isArray(
-          entrypointClass["http://www.w3.org/ns/hydra/core#supportedProperty"]
-        )
-      ) {
-        throw new Error(
-          'The entrypoint definition has no "http://www.w3.org/ns/hydra/core#supportedProperty" key or it is not an array.'
+  return fetchEntrypointAndDocs(entrypointUrl, options)
```

Comment:
A lot of modifications in this file are only `CS` changes. Could you remove them from this PR?